styles/_tutorials.scss

```diff
@@ -0,0 +1,166 @@
+/* Tutorials ---------------------------------------------------------------- */
+.tutorial-content {
+  #beginner-cards {
+    .card:nth-child(1) .card-footer {
+      background-image: url(../img/cards/4col-magenta-3.svg);
+    }
+  }
+
+  #sdk-cards {
+    .card:nth-child(1) .card-footer {
+      background-image: url(../img/cards/4col-light-blue-3.svg);
+    }
+    .card:nth-child(2) .card-footer {
+      background-image: url(../img/cards/4col-magenta-3.svg);
+    }
+    .card:nth-child(3) .card-footer {
+      background-image: url(../img/cards/4col-green-2.svg);
+    }
+    .card:nth-child(4) .card-footer {
+      background-image: url(../img/cards/4col-yellow-2.svg);
+    }
+  }
+
+  #use-cases-cards {
+    .card:nth-child(1) .card-footer {
+      background-image: url(../img/cards/3col-green-purple.svg);
+    }
+    .card:nth-child(2) .card-footer {
+      background-image: url(../img/cards/3-col-light-blue-2.svg);
+    }
+  }
+
+  #servers-cards {
+    .card:nth-child(1) .card-footer {
+      background-image: url(../img/cards/4col-orange-yellow.svg);
```

Comment:
Some use "3-col" named images and others use 4 col for the same set.  That seems a bit odd.

Reply:
I don't really know the distinction between 3col and 4col for the svg colors. Just tried to match the design. Any ideas?